### DISCUSSION TITLE
Providing a Helm chart

### DISCUSSION THREAD

**Discussion by tomkerkhove** [open-telemetry/opentelemetry-collector#4566] [Ideas]
Are there plans to provide a Helm chart to simplify collector installations?

**Comment by jpkrohling** [comment]
I believe there's one already: https://github.com/open-telemetry/opentelemetry-helm-charts

**Reply by tomkerkhove** [reply]
🤦‍♂️ I was checking https://opentelemetry.io/docs/collector/getting-started/#kubernetes & Artifact Hub but couldn't find it.

Might be good to document it there as well.

**Reply by tomkerkhove** [reply]
https://github.com/open-telemetry/opentelemetry-helm-charts/issues/108

**Reply by tomkerkhove** [reply]
https://github.com/open-telemetry/opentelemetry.io/issues/993
